data/categories.txt

```diff
@@ -646,16 +646,16 @@ el:Αρτοποιείο|κατάστημα
 sk:3Pekáreň|obchod
 sw:Kituo cha kuwoka mikate
 
-shop
+shop|@shop
```

Comment:
Or, at least, move prefix-length specifiers to the '@shop' and get rid of duplicates here.